data/com.endlessm.HackToolbox.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.endlessm.HackToolbox.desktop</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LGPL-2.1</project_license>
+  <categories>
+    <category>Game</category>
+  </categories>
+  <name>Toolbox</name>
+  <summary>The other side of flip-to-hack</summary>
+
+  <description><p></p></description>
+
+  <url type="homepage">https://hack-computer.com/</url>
+
+  <releases>
+    <release date="2019-03-25" version="0.0.0"/>
+  </releases>
```

Comment:
I'd also not add this unless you intend to keep it up to date. It's of course harmless, so it's up to you.